Formula/glibc.rb

```diff
@@ -94,42 +94,51 @@ class Glibc < Formula
   end
 
   depends_on "binutils" => :build
+  depends_on "bootstrap-bison" => :build
+  depends_on "bootstrap-gcc" => :build
+  depends_on "bootstrap-make" => :build
+  depends_on "bootstrap-python3" => :build
   depends_on GawkRequirement => :build
-  depends_on "linux-headers@4.4" => :build
+  depends_on "linux-headers@5.15" => :build
```

Comment:
Should this be a runtime dependency?
That was the suggestion of @SMillerDev & @danielnachun in https://github.com/orgs/Homebrew/discussions/3462 & https://github.com/Homebrew/homebrew-core/pull/106680:
"users who need brewed glibc are very likely to also need newer Linux headers"

Reply:
I changed this now to make `linux-headers` a run time dependency, so this problem will be resolved once we've shipped this PR.